File: CursoCsharpUdemy/exer/udemy/EstruturaSequencial/Exer01.cs
using System;

namespace exer.udemy.EstruturaSequencial
{
    public class Exer01
    {
        static void Main(string[] args)
        {
            Console.Write("Digite um número inteiro: ");
            int number1 = int.Parse(Console.ReadLine());

            Console.Write("Digite outro número inteiro: ");
            int number2 = int.Parse(Console.ReadLine());

            int result = number1 + number2;

            Console.WriteLine($"SOMA = {result}");
        }
    }
}

File: Curso de Java/Repeticao-Sequencial-Condicional/src/Exer/EstruturaSequencial/Exer01.java
package Exer.EstruturaSequencial;
import java.util.Scanner;

public class Exer01 {

	public static void main(String[] args) {
		
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Digite o primeiro número: ");
		int x = sc.nextInt();
		System.out.print("Digite o segundo número: ");
		int y = sc.nextInt();
		
		int soma = x + y;
		
		System.out.println("A soma dos numeros é: " + soma);
		
		
		sc.close();
		
	}

}
